pkg/namespacestore/validation.go

```diff
@@ -0,0 +1,36 @@
+package namespacestore
+
+import (
+	"fmt"
+	"strings"
+
+	nbv1 "github.com/noobaa/noobaa-operator/v5/pkg/apis/noobaa/v1alpha1"
+	"github.com/noobaa/noobaa-operator/v5/pkg/util"
+)
+
+// ValidateNamespaceStore validates namespacestore configuration
+func ValidateNamespaceStore(nsStore *nbv1.NamespaceStore) error{
+
+	//pvcName validation
+	if nsStore.Spec.NSFS != nil && nsStore.Spec.NSFS.PvcName == "" {
```

Comment:
instead of checking `nsStore.Spec.NSFS != nil` twice you can check it at the beginning of the function